admission-webhook/deploy/gmsa-crd.yml

```diff
@@ -3,7 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   name: gmsacredentialspecs.windows.k8s.io
   annotations:
-    "api-approved.kubernetes.io": "https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/689-windows-gmsa"
+    "api-approved.kubernetes.io": "https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/689-admission-webhook"
```

Comment:
this should stay as is.  This is supposed to be a link to the [kep where this was approved](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2337-k8s.io-group-protection/README.md#summary).  I don't know that we should be using the k8s.io CRD group here but this was how it was initially implemented and since it is in the k8s.io namespace we need this annotation.

Reply:
Looks like Jetbrains refactoring tool caused these. Thanks for catching them.